src/main/java/org/jenkinsci/plugins/fodupload/controllers/StaticScanController.java

```diff
@@ -94,11 +90,6 @@ public boolean startStaticScan(final JobModel uploadRequest, final String notes)
                     .addQueryParameter("technologyStack", token.getTechnologyType())
                     .addQueryParameter("entitlementId", Integer.toString(assessmentType.getEntitlementId()))
                     .addQueryParameter("entitlementFrequencyType", Integer.toString(assessmentType.getFrequencyTypeId()))
-                    .addQueryParameter("isBundledAssessment", Boolean.toString(assessmentType.isBundledAssessment()))
-                    .addQueryParameter("doSonatypeScan", Boolean.toString(includeOpenSourceScan))
```

Comment:
Was this removed for 314001?  It doesn't seem related.

Reply:
It was removed in another ticket. I had 2 or 3 PRs open at the same time.

Reply:
Didn't realize the merge is going from master to a branch.  This is fine.  Advanced options were removed elsewhere.